src/scicat_kafka.py

```diff
@@ -75,38 +75,41 @@ def validate_consumer(consumer: Consumer, logger: logging.Logger) -> bool:
         return True
 
 
-def _validate_data_type(message_content: bytes, logger: logging.Logger) -> bool:
-    logger.info("Data type: %s", (data_type := message_content[4:8]))
-    if data_type == WRDN_FILE_IDENTIFIER:
+def _validate_wrdn_message_type(message_content: bytes, logger: logging.Logger) -> bool:
+    logger.info("Message type: %s", (message_type := message_content[4:8]))
+    if message_type == WRDN_FILE_IDENTIFIER:
         logger.info("WRDN message received.")
         return True
     else:
-        logger.error("Unexpected data type: %s", data_type)
+        logger.info("Message of type %s ignored", message_type)
```

Comment:
So we're expecting other types of message under this topic...?

Reply:
Correct!!!
We should expect other types of messages.